### PR TITLE
chore: update Adonis to 5.0.2-beta-rc-7

### PR DESCRIPTION
**Pull request by Xstoudi**
@targos , I need you to check if I updated it correctly, especially concerning the logger.

### CODE REVIEW

**Comment by targos** [comment]
The logger in commands is not the same as in applications. Its type should stay as comming from fancy-logs.

File: package.json [hunk 1]
```diff
@@ -69,14 +69,14 @@
...
+    "@adonisjs/ace": "^7.1.5",
+    "@adonisjs/fold": "^7.0.7"
   },
   "devDependencies": {
-    "@adonisjs/ace": "^6.8.0",
-    "@adonisjs/fold": "^6.3.0",
-    "@adonisjs/logger": "^3.0.1",
+    "@adonisjs/core": "^5.0.2-beta-rc-7",
+    "@adonisjs/ace": "^7.1.5",
+    "@adonisjs/fold": "^7.0.7",
```
Comment (targos) [inline package.json:78]: is this still used somewhere? virk told me we should be able to remove this dependency

File: providers/MongodbProvider.ts [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
   public register(): void {
-    this.$container.singleton('Mongodb/Database', () => {
-      const config = this.$container
+    this.app.container.singleton('Mongodb/Database', () => {
+      const config = this.app.container
         .use('Adonis/Core/Config')
         .get('mongodb', {});
-      const Logger = this.$container.use('Adonis/Core/Logger');
+      const Logger = this.app.container.use('Adonis/Core/Logger');
       return new Mongodb(config, Logger);
```
Comment (targos) [inline providers/MongodbProvider.ts:19]: here you can use `this.app.config` and `this.app.logger`

File: tsconfig.json [hunk 3]
```diff
@@ -19,5 +19,8 @@
     "./src/**/*",
     "./providers/**/*",
     "./test-utils/*"
+  ],
+  "files": [
+    "./node_modules/@adonisjs/core/build/adonis-typings/index.d.ts"
   ]
```
Comment (targos) [inline tsconfig.json:25]:
```suggestion
  "types": ["@adonisjs/core", "@types/jest"]
```

File: providers/MongodbProvider.ts (hunk 2 above)
Comment (Xstoudi) [reply providers/MongodbProvider.ts:19]: `this.app.logger` is a LoggerContract, type that is not compatible with `@poppinss/fancy-logs` Logger.
Moreover, `@poppinss/fancy-logs`'s Logger is not the same as `@poppinss/cliui`'s one.

File: tsconfig.json (hunk 3 above)
Comment (Xstoudi) [reply tsconfig.json:25]: `Cannot find module '@ioc:Adonis/Core/Application' or its corresponding type declarations.ts(2307)`

**Review by targos — APPROVED** [review]

**Pull request merged**